### ISSUE TITLE
Please update the following components: withAvatarConfig

### ISSUE THREAD

**Issue by shaheemMPM** [Sitebase/react-avatar#199]
I'm getting the following warning while using react-avatar

```
Warning: Legacy context API has been detected within a strict-mode tree.

The old API will be supported in all 16.x releases, but applications using it should migrate to the new version.

Please update the following components: withAvatarConfig

Learn more about this warning here: https://fb.me/react-legacy-context
    in withAvatarConfig (at Home.js:49)
```

Does anybody know what's causing the issue?

**Comment by JorgenEvens** [comment]
Hi @shaheemMPM,

We provide a fallback for older versiins of React. It is this fallback that is causing this warning. 

It is safe to ignore, see https://github.com/Sitebase/react-avatar/issues/186 for more info.

**Comment by JorgenEvens** [comment]
Hi @shaheemMPM,

I've just released version [v3.9.4](https://github.com/Sitebase/react-avatar/releases/tag/v3.9.4) which only applies the legacy API after detecting whether the new context is available. The React warnings you've reported should no longer be shown after upgrading `react-avatar`.
